packages/taquito/src/context.ts

```diff
@@ -166,6 +166,28 @@ export class Context {
     }
   }
 
+  async getConfirmationPollingInterval() {
+    try {
+      const constants = await this.rpc.getConstants();
+      let confirmationPollingInterval = BigNumber.sum(constants.time_between_blocks[0], 
+             constants.delay_per_missing_endorsement!, 
+             Math.max(0, constants.initial_endorsers! - constants.endorsers_per_block));
```

Comment:
One last thing, while debugging earlier, I find out that the value for confirmationPollingIntervalSecond is 11.333333333333334 instead of 10.
The calculus is different from what you shared on slack some time ago. 
Here it is : 30 + 4 + Max(0,-8)) = 34
On slack : 30 + (4*Max(0,-8)) = 30
Which one is correct?

Reply:
Yes you are right it needs to be multiply. I will fix it [here.[](url)](url
![Screen Shot 2021-03-12 at 10 09 27 AM](https://user-images.githubusercontent.com/2622360/114425903-c0525700-9b76-11eb-8fdd-0822fa5d6b3b.png)
)

Reply:
This is fixed now.